webapp/frontend/src/app/app.tsx

```diff
@@ -38,11 +51,20 @@ export const App = () => {
   const lightPatternList = Object.values(animations).sort((a, b) => a.position - b.position);
 
   return (
-    <div className="bg-gray-200 p-4 pt-16">
+    <div className="bg-gray-200 p-4">
       <ErrorMessage />
+      <p className="leading-tight text-center max-w-fit mx-auto text-5xl font-extrabold text-transparent bg-clip-text text-center bg-gradient-to-r from-rose-600 to-green-600">
```

Comment:
Maybe we should put some of these larger class names in const variables so this is more readable. Something to think about, but I don't care too much either way.

Reply:
Definitely something to consider. I think in general long hard-to-parse class names is a common downside of tailwind. Although the tailwind devs would probably argue that this is more readable than a big hierarchy of custom css classes haha